include/openssl/cmserr.h

```diff
@@ -34,6 +34,7 @@
 #  define CMS_R_CIPHER_PARAMETER_INITIALISATION_ERROR      102
 #  define CMS_R_CMS_DATAFINAL_ERROR                        103
 #  define CMS_R_CMS_LIB                                    104
+#  define CMS_R_OPERATION_UNSUPPORTED                      182
```

Comment:
And here

Reply:
@zuiderkwast don't forget to adapt the order also here.